applications/openshift/kubelet/kubelet_enable_client_server_rotation/rule.yml

```diff
@@ -6,7 +6,9 @@ description: |-
     To enable the kubelet to rotate server certificates, edit the kubelet configuration
     file <tt>/etc/origin/node/node-config.yaml</tt>
     on the kubelet node(s) and set the below parameter:
-    <pre>RotateKubeletServerCertificate=true</pre>
+    <pre>kubeletArguments:
+      feature-gates:
+        - RotateKubeletClientCertificate=true,RotateKubeletServerCertificate=true</pre>
```

Comment:
should this be only for client?

Reply:
Came up with a different description that resolved this questions.